.circleci/config.yml

```diff
@@ -43,17 +43,31 @@ jobs:
       - checkout
       - setup_remote_docker
       - run: docker swarm init
-      - run: docker build -t sleep docker-images/sleep/
-      - run: docker build -t http-server docker-images/http-server/
       - <<: *restore_go_mod_cache
       - run: go mod download
       - <<: *save_go_mod_cache
       - run: mkdir -p ~/.mesg
       - <<: *restore_go_cache
-      - run: go test -v -timeout 300s -p 1 -tags=integration -coverprofile=coverage.txt ./...
+      - run: go test -v -timeout 300s -coverprofile=coverage.txt ./...
```

Comment:
maybe we could even lower the timeout (but this might be a test for another time)

Reply:
yes we can. all the test takes less that 8sec :)

Reply:
but i'm lazy to change it...